### ISSUE TITLE
Question about primary-name disambiguation

### ISSUE THREAD

**Issue by jgm** [citation-style-language/test-suite#40]
I'm getting unexpected results with chicago-author-date.csl, which has  `givenname-disambiguation-rule="primary-name"`.  I think I must be misunderstanding the spec.

The spec says:

> “primary-name-with-initials”
> As “all-names-with-initials”, but disambiguation is limited to the first name of each cite.

Okay, I look that up:

> “all-names-with-initials”
> As “all-names”, but name expansion is limited to showing initials (see step 1(a) above). No disambiguation attempt is made when initialize-with is not set or when initialize is set to “false”.

Now I lookup "all-names":

> “all-names”
> Name expansion has the dual purpose of disambiguating cites and names. All rendered ambiguous names, **in both ambiguous and unambiguous cites**, are subject to disambiguation. Each name is progressively transformed until it is disambiguated. Names that cannot be disambiguated remain in their original form.

So my understanding was that, for ALL of these modes, disambiguation is triggered not just by ambiguous citations but by ambiguous names.

But that's not how citeproc-js seems to work: for example with
```
[
  {
    "author": [
      {
        "family": "Doe",
        "given": "Ann"
      },
      {
        "family": "Doe",
        "given": "Ben"
      },
      {
        "family": "Roe",
        "given": "Ron"
      }
    ],
    "id": "a",
    "issued": {
      "date-parts": [
        [
          2007
        ]
      ]
    }
]
```
we get "Doe, Doe, and Roe 2007" instead of "A. Doe, Doe, and Roe 2007."

What have I missed? (And let me know if there's a more appropriate place to be asking this question.)

**Comment by bdarcus** [comment]
I think it's fine to put the questions here. We can always move the issue to the documentation repo if the problem with the spec.

I've assigned this to Frank and Sebastian, and copying @cormacrelf if he has any thoughts.

We need to remember, BTW, to add comments to tests like this to clarify the expectations, once we settle the issue.

**Comment by adam3smith** [comment]
Primary name only looks at first authors for ambiguity, i.e. you only get
A. Doe of there's another cite with B Doe as first author

**Comment by jgm** [comment]
Thanks, that makes sense.
